### ISSUE TITLE
SSL error when using version >7

### ISSUE THREAD

**Issue by jaedb** [mopidy/infrastructure#2]
Automated GitHub Actions builds are failing with the following SSL error:

<img width="1884" alt="image" src="https://github.com/mopidy/infrastructure/assets/1711444/2e8aef0e-419b-46b6-8a7f-3db27877161e">

When I downgrade from `:latest` to `ghcr.io/mopidy/ci:7` then all works fine:

<img width="1884" alt="image" src="https://github.com/mopidy/infrastructure/assets/1711444/9e09af99-5b11-4c6b-a345-babc0ce24a14">

Has something changed in those more recent releases (~2 months ago) that could have caused this? Or have I got something else I need to upgrade to be compatible with the newer images?

**Comment by jodal** [comment]
Yes, the base image has been changed from `ubuntu:20.04` to `ubuntu:22.04`. You can see the full diff between `7` and `latest` (currently `10`) here: https://github.com/mopidy/infrastructure/compare/ci/7...ci/10

Could you provide a link to the repo or the workflow file? First of, I'm wondering what Python version you set up in the `setup-python` step.

**Comment by kingosticks** [comment]
And we deprecated a python version, if that's relevant.

**Comment by jodal** [comment]
> *This was generated by AI during triage.*

The images after version 7 changed the Ubuntu base image and dropped an old Python version, as explained above. There has been no activity since 2023, and the images have been updated many times since then. Closing. Please open a new issue if the current `ghcr.io/mopidy/ci` image fails.
